docs/flink-getting-started.md

```diff
@@ -634,6 +634,26 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+### Streaming upsert write
+
+Iceberg supports `UPSERT` based on the primary key when writing streaming data into v2 table format. To enable it, create an `UPSERT` table or configuration it when building a writer.
+
+For an `UPSERT` table, all writing will be in `UPSERT` mode unless you overwrite it when building the writer:
+
+```
+CREATE TABLE `hive_catalog`.`default`.`sample` (
```

Comment:
If we want to be consistent with the other two sub sections `append` and `overwrite` just before this, the example should be DataStream code snippet to set the upsert write option. SQL is in a separate section somewhere above.

Reply:
I added upsert doc to the sql and stream sections respectively.